src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -377,6 +377,11 @@ final class SecureStore: SecureKeyValueStoring, Store {
 		get { kvStore["recentTraceLocationCheckedInto"] as DMRecentTraceLocationCheckedInto? ?? nil }
 		set { kvStore["recentTraceLocationCheckedInto"] = newValue }
 	}
+	
+	var hibernationComparingDate: Date {
```

Comment:
I think we should the if !RELEASE here and not in the initializer for the CWAHibernationProvider, we already did this several times, for example: elsLoggingActiveAtStartup in SecureStore

Reply:
Please see line `354.` `hibernationComparingDate` is in the `!RELEASE` block. ;)